.gitattributes

```diff
@@ -59,6 +59,7 @@
 *.h text
 *.h++ text
 *.hh text
+*.qml text
```

Comment:
We also need `.pro`, `.pri` and `.qrc` files, and maybe the `.sh` files too.  Not sure if that's all...